big-red-shuttle/GPS.swift

```diff
@@ -107,15 +116,15 @@ import SwiftyJSON
     }
     
     func notifyObserversGPSBeganLogging() {
-        let _ = self.observers.map({ $0?.gpsBeganLogging?(gps: self) })
+        let _ = self.observers.map({ $0?.gpsStartedLogging?(gps: self) })
```

Comment:
They have already addressed it, it's called a for loop. They removed the foreach functional call a while ago.